src/controllers/user.controller.ts

```diff
@@ -1,6 +1,9 @@
-import { addUserRequestSchema, getUserRequestSchema } from "../models/user";
+import { addUserRequestSchema, getUserRequestSchema, loginUserRequestSchema } from "../models/user";
 import { userRepository } from "../repositories/user.repository";
 import { HttpResponse } from "../models/httpResponse";
+import { tokenService } from "../services/token.service";
+import bcrypt from "bcrypt";
+import "dotenv/config";
```

Comment:
I think this should be imported at the most top-level file (where the express server is created)